inc/utility/namespace.php

```diff
@@ -43,12 +43,14 @@ function scandir_by_date( $dir ) {
 
 	$ignored = [ '.', '..', '.svn', '.git', '.htaccess' ];
 
+	$dir = rtrim( $dir, '/' ) . '/';
```

Comment:
Nitpicking at this point but we prefer `untrailingslashit()` found `wp-includes/formatting.php`

`$dir = untrailingslashit( $dir ) . '/';`

Because `\this\is\a\valid/path` on Windows.

Reply:
Ah, I see. I was just copying [group_exports](https://github.com/eSkript/pressbooks/blob/a16470c9cb46ed78d1837e44f2d5fa191b37cd16/inc/utility/namespace.php#L76). So if you ever want to adapt it, there is an other instance.